src/pages/about-greenwashing.tsx

```diff
@@ -0,0 +1,7 @@
+
+function AboutGreenwashing() {
+  return <h2>Comprendre le Greenwashing</h2>;
```

Comment:
~~plutôt h1 si c'est le premier titre? ([mieux pour l'accessibilité](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/))~~ my bad, y'a déjà un h1 un peu plus haut